source/common/http/conn_manager_impl.cc

```diff
@@ -388,6 +388,10 @@ uint64_t ConnectionManagerImpl::ActiveStream::connectionId() {
   return connection_manager_.read_callbacks_->connection().id();
 }
 
+const Network::Connection* ConnectionManagerImpl::ActiveStream::connection() {
+  return &connection_manager_.read_callbacks_->connection();
+}
+
 Ssl::Connection* ConnectionManagerImpl::ActiveStream::ssl() {
   return connection_manager_.read_callbacks_->connection().ssl();
```

Comment:
It seems like we wouldn't have the `ssl()` method on this interface if starting from scratch, since we could get it via `connection()`. Thoughts on whether this should be deprecated? @mattklein123 as well.